kloppy/infra/serializers/event/opta/deserializer.py

```diff
@@ -761,6 +792,14 @@ def deserialize(self, inputs: OptaInputs) -> EventDataset:
                             **duel_event_kwargs,
                             **generic_event_kwargs,
                         )
+                    elif type_id == EVENT_TYPE_INTERCEPTION:
```

Comment:
Opta also has a "blocked pass" event. Would it make sense to parse it as an interception too?

blocked pass := When a player tries to cut out an opposition pass by any means. Similar to an interception except there is much less reading of the pass.